conf/task_defaults/chesapeake_cvpr_7.yaml

```diff
@@ -20,8 +21,8 @@ experiment:
     - "de-test"
     test_splits:
     - "de-test"
-    patches_per_tile: 200
-    patch_size: 256
-    batch_size: 64
+    patches_per_tile: 2
+    patch_size: 64
+    batch_size: 2
```

Comment:
BYOLTask tests fail with a batch size of 1 and I have no idea why. SemanticSegmentationTask tests work fine with a batch size of 1. Oh, the mysteries of life...

Reply:
If any preprocessing methods use `.squeeze()`, then they will remove the batch dimension which will in turn break the forward pass

Reply:
We should update these to define the only dim that should be squeezed like `.squeeze(dim=1)`

Reply:
Just tried this (both dim=0 and dim=1) and as soon as it fixes one issue, it creates another one. I don't think I have the time to debug this any further, but if anyone wants to submit a follow-up PR to fix this I would be very happy.